saskatoon/fixtures/loaddata

```diff
@@ -22,9 +22,9 @@ if [ $1 == "all" ]; then
     for f in $(ls *.json | grep -v "^saskatoon")
     do
         echo "Loading $f ..."
-        ../manage.py loaddata "${f%.*}"
+        ../manage.py loaddata "${f%.*}" || exit 1
```

Comment:
FYI in `saskatoon/fixtures/init` I used `set -e` at the top of the script to automatically exit on error, maybe that could help here

Reply:
Yes both should work, but better safe.